docs/source/diagrams/catchup-procedure.puml

```diff
@@ -305,36 +301,6 @@ loop For each replica
 end loop
 
```

Comment:
Instead of `primaries = primaries from last audit txn` we need to say
- node_reg is recovered from the audit ledger
- primaries are calculated based on recovered node registry (using a node registry as it was at the beginning of previous view).